mapview/__init__.py

```diff
@@ -16,7 +20,12 @@
 MAX_LATITUDE = 90.
 MIN_LONGITUDE = -180.
 MAX_LONGITUDE = 180.
-CACHE_DIR = "cache"
+if platform == 'ios':
+    # CACHE_DIR must be in a place where the user has read/write permissions
+    root_folder = App().user_data_dir
```

Comment:
Is it possible to avoid creating a new App() object here? maybe first try to use App.get_running_app() instead, especially since `App._get_user_data_dir` seems to depend on the app name for ios.

Reply:
Oops, that's definitely what I meant to do.

Reply:
Hmm, at the time the module is loaded, App.get_running_app() returns None, so we can't use that to get the user_data_dir. Will try to think of another fix but chime in if you have one in mind.

Reply:
hm, maybe the cache dir could be computed the first time it's needed, and reused after, using a reified method, instead of being a global then?

Reply:
How about seeing if the platform is iOS during MapView's `__init__`? Seems to work fine

Reply:
seems like a better idea, see my comment for a minor correction, but aside that seems good to me.